src/ServiceProviders/LegacyServiceProvider.php

```diff
@@ -24,7 +25,6 @@ public function register() {
 	 * @inheritDoc
 	 */
 	public function boot() {
```

Comment:
I know it needs to method to match the interface, but maybe add a comment like "this section intentionally left blank" or point to where the `register_activation_hook` was moved.

Not a blocker.

Reply:
It's not really obvious that this is where the activation hook was registered previously. In the future, I'd like to add an abstract `ServiceProvider` class so we can remove the empty functions.